universe/node/api_object.go

```diff
@@ -468,12 +468,39 @@ func (n *Node) apiCloneObject(c *gin.Context) {
 				defaultValue := utils.GetFromAnyMap(cloneable, "use_default", map[string]any(nil))
 				if defaultValue != nil {
 					attributeValue := entry.AttributeValue(defaultValue)
-					cloneableObjectAttribute := CloneableObjectAttribute{
+					cloneableDefaultAttribute := CloneableObjectAttribute{
 						AttributeID: attributeID,
 						Value:       &attributeValue,
 					}
 
-					cloneableAttributes = append(cloneableAttributes, cloneableObjectAttribute)
+					cloneableAttributes = append(cloneableAttributes, cloneableDefaultAttribute)
+				} else {
+					objectColorAttributeID := entry.NewAttributeID(universe.GetSystemPluginID(), "object_color")
```

Comment:
the point of having clonable in option is so we don't have to hardcode the attributes (and can easily add new ones that need to be cloned).
So instead of getting "object_color" and "state" here, I think we can grab the value from the range loop we are in here `objectAttributes[attributeID]`